.github/ISSUE_TEMPLATES/2-feature-request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Request a new feature for the Props Bot.
+labels: ['[Type] Feature Request']
```

Comment:
this doesn't match the current `enhancement` label, so either update here or update the repo label